cfgreader.go

```diff
@@ -68,13 +70,16 @@ func cfgReader(cmd *CmdCtx, cCtx *cli.Context, forceProdUrl bool) (azblob.Reader
 	}
 
 	if envAuth {
-		reader, err = azblob.NewDev(azblob.NewDevConfigFromEnv(), container)
+		devCfg := azblob.NewDevConfigFromEnv()
```

Comment:
We might be able to rename this to indicate that its about privileged / direct access